### ISSUE TITLE
binary not working with gpthd

### ISSUE THREAD

**Issue by martinsteih** [Linutronix/elbe#306]
I tried to insert a binary command within gpthd, but XML validation tells me it's an unexpected command, though it works within msdoshd.

**Comment by bgermann** [comment]
That is right, it is missing in the schema for GPT. Would that be a correct use case?

**Comment by martinsteih** [comment]
Yes, that’s my use case. I just want to copy my u-Boot binary to a specific offset of my first partition. Exactly like the example with msdoshd except that I need gpthd. 
Curiously, I did add it to the Schema but Elbe still complained with „unexpected …“ but I‘ll double check it once again.

**Comment by martinsteih** [comment]
I copied the binary part of msdoshd to gpthd in the schema, but still, it does not work. Within the code, I'm unable to find anything that might not work, so I am a little confused.

**Comment by bgermann** [comment]
If you have elbe installed, the schema is not taken from git but from the system's xml catalogue.

**Comment by martinsteih** [comment]
Still the same. I replaced every occurrence of the schema and even uninstalled elbe to make sure my repo clone is the only possible source of anything. Still, I get the same error. I even double-checked the file itself. As far as I see it, there is only one occurrence of gphd that is to be edited. Next, to be absolutely certain, I removed my initvm and created a new one. But the error still persists. Any hints?
